### PR TITLE
[SW-2369] Copy Sparkling Water booklet from H2O-3 repo to Sparkling Water

### PR DESCRIPTION
**Pull request by jakubhava**
This change just makes the Sparkling Water booklet part of our infrastructure. The content is just copied from h2o-3 repo without any changes. The updates to the content will come in following PRs (so I would not review the content of tex files at this stage)

### CODE REVIEW

**Comment by mn-mikke** [comment]
What are the benefits of having this booklet? If this material contains extra pieces of information, shouldn't these pieces be moved to SW documentation everyone could search on it?

**Comment by jakubhava** [comment]
We give out these booklets on conferences etc, so we need to keep them. 

I'm moving them from H2O-3 repo since they have been stored there for historical reasons so they are easier to maintain on our side

File: booklet/src/sections/pysparkling.tex [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+Launch a local Spark cluster.
+\begin{lstlisting}[style=Bash]
+export MASTER="local[*]"
+\end{lstlisting}
+
+\item From your terminal, run:
+
+\begin{lstlisting}[style=Bash]
+cd ~/Downloads
+unzip sparkling-water-2.2.2.zip
```
Comment (mn-mikke) [inline booklet/src/sections/pysparkling.tex:34]: This is too old...

File: booklet/src/sections/pysparkling.tex [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+Once the H2O cluster is up, the H2O-Python package is used to interact with the cluster and run H2O algorithms. All pure H2O calls are executed via H2O's REST API interface. Users can easily integrate their regular PySpark workflow with H2O algorithms using PySparkling Water.
+
+PySparkling Water programs can be launched as an application, or in an interactive shell, or notebook environment.
+
+\subsection{Getting Started:}
+
+\begin{enumerate}
+\item Download Spark (if not already installed) from the Spark Downloads Page.
+
+Choose Spark release : 2.2.0
```
Comment (mn-mikke) [inline booklet/src/sections/pysparkling.tex:15]: Spark version could be more up to date

File: booklet/src/sections/backends.tex [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+    .setYARNQueue("h2o_yarn_queue")
+val hc = H2OContext.getOrCreate(spark, conf)
+\end{lstlisting}
+
+
+and in Python:
+\begin{lstlisting}[style=Python]
+from pysparkling import *
+conf = H2OConf(spark)
+    .set_external_cluster_mode()
```
Comment (mn-mikke) [inline booklet/src/sections/backends.tex:192]: Old syntax

File: booklet/src/sections/backends.tex [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+    .setClientIp("192.168.0.3")
+    .setCloudName("test")
+val hc = H2OContext.getOrCreate(spark, conf)
+\end{lstlisting}
+
+Python:
+\begin{lstlisting}[style=Python]
+from pysparkling import *
+conf = H2OConf(spark)
+    .set_external_cluster_mode()
```
Comment (mn-mikke) [inline booklet/src/sections/backends.tex:156]: Old api

File: booklet/src/sections/backends.tex [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+    .use_manual_cluster_start()
+    .set_h2o_cluster("ip", port)
+    .set_client_ip("192.168.0.3")
+    .set_cloud_name("test")
+hc = H2OContext.getOrCreate(spark, conf)
+\end{lstlisting}
+
```
Comment (mn-mikke) [inline booklet/src/sections/backends.tex:163]: Missing R example

File: booklet/src/sections/apps.tex [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+
+This is a structure of a simple example project to start coding with Sparkling Water. The source code is available at
+\url{https://github.com/h2oai/h2o-droplets/tree/master/sparkling-water-droplet}
+
+\textbf{Dependencies}
+
+This droplet uses Sparkling Water 2.2, which integrates:
+\begin{itemize}
+\item Spark 2.2
+\item H2O 3.14.0.7 Weierstrass
```
Comment (mn-mikke) [inline booklet/src/sections/apps.tex:13]: Old versinos

**Comment by jakubhava** [comment]
@mn-mikke thanks for review, but as mentioned above in the description, I haven't updated the booklet in this PR. Hence I will skip your comments in this PR. Thanks anyways!

**Pull request merged**